ionic2/app/pages/map/map.page.html

```diff
@@ -10,4 +10,7 @@
   </button>
 
   <poke-poi-card #poiCard></poke-poi-card>
+
+  <button id="artificial-mob-button"
```

Comment:
- I can remove it immediately, e.g. by uncommenting it
- Or keep it for a while, e.g. remove it before we launch/map shows mobs